config/default.yml

```
@@ -282,8 +282,15 @@ Rails/FindEach:
   StyleGuide: 'https://rails.rubystyle.guide#find-each'
   Enabled: true
   VersionAdded: '0.30'
+  VersionChanged: '2.8.2'
```

Can you update to `'2.9'`?

```suggestion
  VersionChanged: '2.9'
```